draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -242,11 +242,43 @@ At a high level, the Verifier
 
 The detailed algorithm is described in (#verifier_verification).
 
-# Data Formats {#data_formats}
+# SD-JWT Data Formats {#data_formats}
 
-This section defines data formats for SD-JWT including the Issuer-signed JWT content, Disclosures, and Key Binding JWT.
+An SD-JWT is composed of the following:
+
+* an Issuer-signed JWT,
+* zero or more Disclosures, and
+* optionally a Key Binding JWT.
+
+The individual parts will be explained in the following subsections.
+
+The serialized format for the SD-JWT is the concatenation of each part delineated with a single tilde ('~') character as follows:
+
+```
+<Issuer-signed JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>~<optional KB-JWT>
+
+```
+
+The order of the tilde separated values MUST be the Issuer-signed JWT, followed by any number of Disclosures, and lastly the optional Key Binding JWT.
+In the case that there is no Key Binding JWT, the last element MUST be an empty string and the last separating tilde character MUST NOT be omitted.
 
-## SD-JWT Payload
+The Disclosures are linked to the SD-JWT payload through the
+digest values included therein.
+
+When issued to a Holder, the Issuer includes all the relevant Disclosures in the SD-JWT.
+
+For presentation to a Verifier, the Holder sends the SD-JWT including only its selected
+set of the Disclosures to the Verifier.
+
+The Holder MAY send any subset of the Disclosures to the Verifier, i.e.,
+none, multiple, or all Disclosures. For data that the Holder does not want to reveal
```

Comment:
```suggestion
none, parts of, or all Disclosures. For data that the Holder does not want to reveal
```
`parts of ` feels a little off, but `all` is part of `multiple`..

Reply:
`some of` ?

Reply:
```suggestion
none, some, or all Disclosures. For data that the Holder does not want to reveal
```
WDYT?

Reply:
WFM